test-desktop/debian/bookworm/entrypoint.sh

```diff
@@ -9,7 +9,7 @@ TARGET_PACKAGE=$3
 apt update
 
 # Install repo key
-wget -qO - "$APT_KEY_URL" | apt-key add -
+wget -qO - "$APT_KEY_URL" | sudo tee /etc/apt/keyrings/regolith.gpg
```

Comment:
`gpg --dearmor` is missing before piping back to `tee`

Reply:
ah nice, I didn't notice that in the diff from the earlier PR.  Fixed in the subsequent commit.